addons/google_calendar/models/google_sync.py

```diff
@@ -118,7 +118,7 @@ def _from_google_ids(self, google_ids):
             return self.browse()
         return self.search([('google_id', 'in', google_ids)])
 
-    def _sync_odoo2google(self, google_service: GoogleCalendarService):
+    def _sync_odoo2google(self, google_service: GoogleCalendarService, full_sync=False):
```

Comment:
Note: You cannot change a method signature on a stable release, as this will break the potential customizations.

See our stable policy: https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean